src/main/java/se/kth/bbc/project/ProjectsManagementBean.java

```diff
@@ -61,8 +67,14 @@ public void setAllProjects(List<ProjectsManagement> allProjects) {
     return allProjects;
   }
 
-  public int getHdfsQuota() {
-    return projectsManagementController.getHdfsQuota();
+  public int getHdfsQuota(String projectname) throws IOException {
+    long quota = projectsManagementController.getHdfsQuota(projectname);
+    this.hdfsquota = String.valueOf(quota);
+    return (int) quota;
+  }
```

Comment:
Catch NumberFormatException here. Don't want HopsWorks to crash just because somebody put an invalid number in the quota textfield. We should validate that the number being entered is a valid number, too - in the project administrator UI.